### PR TITLE
Removes duplicate scribe implementation

### PR DESCRIPTION
**Pull request by codefromthecrypt**
`zipkin-receiver-scribe` was implented as a replacement for
`zipkin-collector-scribe`, but `zipkin-collector-service` was never
updated to use it. This consolidates towards receiver and deletes the
orphaned code.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
FYI twitter doesn't use any of the affected code.

File: zipkin-collector-service/build.gradle [hunk 1]
```diff
@@ -21,8 +21,7 @@ repositories {
 }
 
 dependencies {
-    compile project(':zipkin-collector-scribe')
-    compile project(':zipkin-receiver-kafka')
```
Comment (codefromthecrypt) [inline zipkin-collector-service/build.gradle:25]: removed receiver-kafka as it wasn't used. can be added back in the future.

**Comment by eirslett** [comment]
+1 :beers:

**Comment by jamescway** [comment]
:+1:

**Pull request merged**